src/LightGraphs.jl

```diff
@@ -7,7 +7,7 @@ using Base.Collections
 using LightXML
 using ParserCombinator: Parsers.DOT, Parsers.GML
 
-import Base: write, ==, <, *, isless, issubset, complement, union, intersect,
+import Base: write, ==, <, *, ≈, isless, issubset, complement, union, intersect,
```

Comment:
> Why are we overriding assignment?

I guess it is readability problem of \approx. Should I rewrite it as below
`import Base: write, ==, <, *, isless, issubset, complement, union, intersect,
            reverse, reverse!, blkdiag, getindex, setindex!, show, print, copy, in,
            sum, size, sparse, eltype, length, ndims, issym, transpose,
            ctranspose, join, start, next, done, eltype, get, ≈ # ≈ (Base.isapprox)`

> I added some line comments. One thing I really appreciate is your documentation. It's very thorough!!

Well, I don't know who wrote the maximum_flow.jl file but it was definitely a motivation for good documentation (and comments)!